### PR TITLE
[ID] Add `Forum`

### PR DESCRIPTION
**Pull request by Nivalyx**
Indonesian translation for the `Forum` (http://osu.ppy.sh/wiki/en/Community/Forum) wiki article.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Community/Forum/id.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+*Lihat juga: [Forum internet](https://id.wikipedia.org/wiki/Forum_internet)*
+
+[**Forum osu!**](https://osu.ppy.sh/community/forums) merupakan sarana percakapan di mana seluruh pesan yang masuk dikelompokkan berdasarkan subjeknya masing-masing. Tidak seperti [osu! chat](/wiki/Community/Internet_Relay_Chat) yang berlangsung secara instan, diskusi pada forum bersifat permanen serta dapat dicari dan dibaca ulang kapan saja. Oleh karenanya, pesan forum yang ada pada umumnya lebih panjang, lebih kaya akan informasi, dan lebih rinci dibanding pesan *chat* biasa.
+
+Forum osu! menawarkan berbagai fitur yang serupa dengan forum internet lain pada umumnya, mulai dari [BBCode](/wiki/BBCode), *markup*, jajak pendapat, pesan pribadi, penyematan dan langganan, hingga [moderasi](#moderasi).
+
+## Struktur
+
+Masing-masing forum mengandung berbagai macam **topik** (atau yang juga dikenal sebagai *utas*) serta kategori spesifik di dalamnya yang disebut dengan **sub-forum**. Hampir seluruh forum mewajibkan penggunaan Bahasa Inggris, namun terdapat juga [forum khusus bahasa tertentu](https://osu.ppy.sh/community/forums#forum-23) bagi komunitas lokal dengan basis pengguna yang aktif.
```
Comment (hikikuman) [inline wiki/Community/Forum/id.md:17]:
```suggestion
Masing-masing forum mengandung berbagai macam **topik** (atau yang juga dikenal sebagai *utas*) serta kategori spesifik di dalamnya yang disebut dengan **sub-forum**. Hampir seluruh forum mewajibkan penggunaan bahasa Inggris, namun terdapat juga [forum khusus bahasa tertentu](https://osu.ppy.sh/community/forums#forum-23) bagi komunitas lokal dengan basis pengguna yang aktif.
```

File: wiki/Community/Forum/id.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+[Moderator](/wiki/People/The_Team/Global_Moderation_Team) merupakan kelompok pengguna yang bertugas menjaga ketertiban dan kenyamanan forum serta menanggapi laporan penyalahgunaan. Para moderator dapat menyunting atau menghapus postingan atau topik yang dianggap tidak pantas, menyematkan dan memindahkan topik dari satu forum ke forum lainnya, serta mencegah para pelanggar untuk dapat berulah lebih lanjut dengan [menjatuhkan *silence*](/wiki/Silence) kepada mereka.
+
+## Forum khusus
+
+Beberapa forum tertentu memiliki fungsinya tersendiri yang tidak terbatas hanya sebagai sarana diskusi. Forum ini pada umumnya memiliki peraturan dan/atau batasan teknis tambahannya tersendiri.
+
+- [Help](/wiki/Community/Forum/Help): forum untuk mengajukan bantuan teknis dari [Support Team](/wiki/People/The_Team/Support_Team) dan para pengguna lainnya.
+- [Feature Requests](/wiki/Community/Forum/Feature_Requests): forum untuk mengajukan saran untuk pengembangan osu!.
+- [Modding Queues](/wiki/Community/Forum/Modding_Queues): forum untuk mengajukan kajian beatmap dari mapper dan modder lain.
```
Comment (hikikuman) [inline wiki/Community/Forum/id.md:39]:
```suggestion
- [Modding Queues](/wiki/Community/Forum/Modding_Queues): forum untuk mengajukan umpan balik (*feedback*) beatmap dari mapper dan modder lain.
```
Penggunaan kata `kajian` terlalu sulit dicerna dan kurang umum di dunia mapping/modding, gimana kalo pake `umpan balik` sebagai gantinya? Soalnya biasanya kita pakai kata `feedback` sebagai kaitan aktivitas modding sebuah beatmap.

File: wiki/Community/Forum/id.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- [Off-Topic](/wiki/Community/Forum/Off-Topic): forum untuk mendiskusikan segala sesuatunya secara serampangan.
+- [Abandoned Wasteland](/wiki/Community/Forum/Abandoned_Wasteland): tempat pembuangan topik yang tidak diinginkan.
+
+Di samping itu, terdapat juga beberapa forum rahasia seperti Abandoned Wasteland lainnya yang ditujukan untuk keperluan teknis atau manajemen.
```
Comment (hikikuman) [inline wiki/Community/Forum/id.md:43]:
```suggestion
Di samping itu, terdapat juga beberapa forum rahasia yang sama seperti Abandoned Wasteland, yang ditujukan untuk keperluan teknis atau manajemen.
```

File: wiki/Community/Forum/id.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+tags:
+  - forums
+  - subforum
+---
+
+# Forum
+
+*Lihat juga: [Forum internet](https://id.wikipedia.org/wiki/Forum_internet)*
+
+[**Forum osu!**](https://osu.ppy.sh/community/forums) merupakan sarana percakapan di mana seluruh pesan yang masuk dikelompokkan berdasarkan subjeknya masing-masing. Tidak seperti [osu! chat](/wiki/Community/Internet_Relay_Chat) yang berlangsung secara instan, diskusi pada forum bersifat permanen serta dapat dicari dan dibaca ulang kapan saja. Oleh karenanya, pesan forum yang ada pada umumnya lebih panjang, lebih kaya akan informasi, dan lebih rinci dibanding pesan *chat* biasa.
```
Comment (hikikuman) [inline wiki/Community/Forum/id.md:11]:
```suggestion
[**Forum osu!**](https://osu.ppy.sh/community/forums) merupakan sarana percakapan di mana seluruh pesan yang masuk dikelompokkan berdasarkan subjeknya masing-masing. Tidak seperti [chat osu!](/wiki/Community/Internet_Relay_Chat) yang berlangsung secara instan, diskusi pada forum bersifat permanen serta dapat dicari dan dibaca ulang kapan saja. Oleh karenanya, pesan forum yang ada pada umumnya lebih panjang, lebih kaya akan informasi, dan lebih rinci dibanding pesan *chat* biasa.
```
Biar sama formatnya dengan yang lain kaya `osu! forum` > `forum osu!`, `osu! website` > `web osu!`, dll.

**Review by hikikuman — CHANGES_REQUESTED** [review]
it might be good to add `forum` and `sub-forum` to the tags in id.md and change the Language Specific link from https://osu.ppy.sh/community/forums#forum-23 to https://osu.ppy.sh/community/forums/23

**Comment by Nivalyx** [comment]
`forum` is quite literally the title of the article already so i don't think it should be added to the tags tho

fixed everything else, should be ready for recheck/approval i suppose @hikikuman 🦉

**Review by hikikuman — DISMISSED** [review]
Alright, then good to go👍

File: wiki/Community/Forum/id.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+---
+tags:
+  - forums
+  - subforum
+  - sub-forum
```
Comment (Walavouchey) [inline wiki/Community/Forum/id.md:5]: copy the new tag to the other ones

**Comment by Nivalyx** [comment]
@Walavouchey done 👍

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**